tests/unit/fake_data_root/storage/ceph-mon/sos_commands/ceph_mon/json_output/ceph_osd_df_tree_--format_json-pretty

```diff
@@ -94,10 +94,10 @@
             "pool_weights": {},
             "reweight": 1,
             "kb": 104837120,
-            "kb_used": 1048984,
+            "kb_used": 2097558,
```

Comment:
I know its a pain but if there is anyway that your test could make these changes on a temporary basis that would be preferable. I prefer to keep the contents of the fakle_data_roots "real" if possible.

Reply:
Try this: https://pastebin.canonical.com/p/fxbCY48BZn/

Reply:
Applied, thanks. Also removed the short 7 full examples from 'functional' test as it no longer works (due to 'fake_data_root' not containing the data to cause the expected failures).